systemtests/src/main/java/io/enmasse/systemtest/selenium/page/ConsoleWebPage.java

```diff
@@ -317,6 +313,54 @@ private WebElement getFilterRegexAlertClose() throws Exception {
         return selenium.getWebElement(() -> selenium.getDriver().findElement(By.className("pficon-close")));
     }
 
+    //
+    //Create address modal window getters
+    //
+
+    public WebElement getAddressModalNextButton() throws Exception {
+        return selenium.getWebElement(() -> selenium.getDriver().findElement(By.id("nextButton")));
+    }
+
+    /**
+     * get Address Modal window Back button
+     */
+    private WebElement getAddressModalBackButton() throws Exception {
```

Comment:
same as above, I implemented these methods long time ago, you don't need to create same methods with different name

Reply:
removed